src/countFactorials.cpp

```diff
@@ -0,0 +1,39 @@
+/*Problem - To find the number of factorials in a given range.
+
+Description - You will be given an input array[l, h] which gives the range.Your job is to return the count of the number of factorials between 'l' and 'h' both included.
+
+Example: input = [2, 10]
+		 The factorails between 2 and 10 is '2' alone.Hence 1 must be returned.
+
+
+Constraints :
+1) The range should not be negative, i.e neither of l and h should be negative.If so, return -1.
+2) l, h are both long integers.
+3) The range should be non decreasing, i.e l <= h.It is not, return -1.
+
+DONT CHANGE THE FUNCTION HEADERS.
+
+
+
+*/
+
+#include <stdio.h>
+
+
+int countFactorials(long l, long h)
+{
+	long a[17] = { 0 };
+
```

Comment:
Why is it 17 here, can you explain?

Reply:
Anna for the question countFactorial I used 17 as array size because after 16! it's value exceed size of an integer. So, to avoid that I used 17.
If in case we are dealing with highest numbers then we can use long type and store highest values.